pkg/synopsysctl/cmd_create.go

```diff
@@ -169,31 +176,28 @@ var createBlackDuckCmd = &cobra.Command{
 		log.Debugf("setting template spec %s", baseBlackDuckSpec)
 		err = createBlackDuckCtl.SwitchSpec(baseBlackDuckSpec)
 		if err != nil {
-			log.Errorf("%s", err)
-			return nil
+			cmd.Help()
+			return fmt.Errorf("%s", err)
```

Comment:
SwitchSpec should be returning an error type, so no need to fmt as errorf.  Just return the err